addons/website/static/src/scss/website.scss

```diff
@@ -363,6 +363,19 @@ font[class*='bg-'] {
     }
 }
 
+// Forms
+.form-check, .form-select, .css_attribute_color {
+    &:not(:disabled):not(.disabled) {
+        &, input, label {
+            cursor: pointer;
+        }
```

Comment:
Annoying, bootstrap did not make good rules about this? I see `$form-check-label-cursor` as a variable? If indeed, we want more, I suggest the same as above: either this should be a frontend-only rule (and would do it in web/frontend.scss) or we judge this is a fix/imp of bootstrap as a whole and it should be in bootstrap_review.scss (probably what I would do).

(note that I did not check what happened to that file with the BS5 tsunami)

Reply:
the variable has been introduced in https://github.com/twbs/bootstrap/pull/29633... but without a lot of enthusiasm I'd say ^^ 
BS's implementation doesn’t handle the `disabled` input scenario for example :/

I'll move it in `bootstrap_review.scss` , thanks